packages/react-components/src/molecules/__tests__/ListControls.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import ListControls from '../ListControls';
+
+it('passes through links', () => {
+  const { getByTitle } = render(
+    <ListControls
+      cardViewHref="/card?123"
+      listViewHref="/list?321"
+      isListView={false}
+    />,
+    { wrapper: MemoryRouter },
+  );
+  expect(getByTitle(/list/i).closest('a')).toHaveAttribute('href', '/list?321');
+  expect(getByTitle(/card/i).closest('a')).toHaveAttribute('href', '/card?123');
+});
+
+it('indicates which option is selected', () => {
+  const { getByText, rerender } = render(
+    <ListControls
+      cardViewHref="/card?123"
+      listViewHref="/list?321"
+      isListView={false}
+    />,
+    { wrapper: MemoryRouter },
+  );
+  expect(
+    getComputedStyle(getByText(/card/i, { selector: 'p' })).fontWeight,
```

Comment:
Avoid `selector` since it's overly specific (we don't need to test that it's in a `p`), and use `findParentWithStyle`

Reply:
The problem was get by text was picking up the SVG title. Thats why i put the selector

Reply:
Ah ok